source/admin_manual/known_issues/large_cache.rst

```diff
@@ -4,6 +4,8 @@
 dovecot.index.cache is too large
 ================================
 
+**Problem type 1.**
```

Comment:
Should this be ------------ title instead?

Reply:
I tried --- but it looked strange on my computer - the headings were of very similar size. Anyway, Ill change it to --- as that is technically more correct.